oct/tournament/serializers.py

```diff
@@ -53,7 +53,12 @@ def _transform(self, obj, fields, exclude, include):
                     include.get(field)
                 )
             else:
-                data[json_name] = value
+                if field_type is None and hasattr(obj, field):
+                    data[json_name] = getattr(obj, field)
+                elif isinstance(value, datetime):
+                    data[json_name] = value.isoformat()
+                else:
+                    data[json_name] = value
```

Comment:
get rid of the first if statement and unnest the other two

Reply:
im probably just stupid, wdym by unnest them

Reply:
the if statements are nested inside the else statement for no reason